happybase/thrift_sasl.py

```diff
@@ -0,0 +1,169 @@
+""" SASL transports for Thrift. """
```

Comment:
This is pretty standard stuff, see for example here https://github.com/cloudera/thrift_sasl/blob/master/thrift_sasl/__init__.py